src/tests/exprtype/exprtype_test.go

```diff
@@ -2707,6 +2707,32 @@ function f1() {
 	runExprTypeTest(t, &exprTypeTestParams{code: code, stubs: "<?php /* no code */"})
 }
 
+func TestInstanceDeserializeType(t *testing.T) {
```

Comment:
Consider to move this test to `kphp_test.go`.

Reply:
Done